persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -167,7 +167,7 @@ SELECT * FROM ConversatonDetails
 WHERE type IS NOT "SELF" AND
 (type IS "GROUP" AND (name IS NOT NULL OR otherUserId IS NOT NULL) --filter deleted groups after first sync
 OR (type IS NOT "GROUP" AND otherUserId IS NOT NULL)) -- show other conversation- todo: problem here! if the sync wasn't succesful the the user seems to be null!
-AND (protocol IS "MLS" AND mls_group_state IS "ESTABLISHED")
+AND (protocol IS "PROTEUS" OR (protocol IS "MLS" AND mls_group_state IS "ESTABLISHED"))
```

Comment:
The where statement look like this:
```
type IS NOT "SELF" 
AND (type IS "GROUP" AND (name IS NOT NULL OR otherUserId IS NOT NULL) OR (type IS NOT "GROUP" AND otherUserId IS NOT NULL))
AND (protocol IS "PROTEUS" OR (protocol IS "MLS" AND mls_group_state IS "ESTABLISHED"))
```

If we change it to:
```
type IS NOT "SELF" 
AND (type IS "GROUP" AND (name IS NOT NULL OR otherUserId IS NOT NULL) OR (type IS NOT "GROUP" AND otherUserId IS NOT NULL))
OR (protocol IS "MLS" AND mls_group_state IS "ESTABLISHED")
```

It would skip 
```
type IS NOT "SELF" 
AND (type IS "GROUP" AND (name IS NOT NULL OR otherUserId IS NOT NULL) OR (type IS NOT "GROUP" AND 
```

if `OR (protocol IS "MLS" AND mls_group_state IS "ESTABLISHED")` evaluates to **true**.

Reply:
Yes, thanks!